client/src/components/main/App/AppMenu.tsx

```diff
@@ -0,0 +1,163 @@
+        transcript: "refseq:NM_002609.3",
+        input_tx: "NM_002609.3",
+        exon_start: 11,
+        exon_start_offset: 0,
+        gene_descriptor: {
+          id: "normalize.gene:PDGFRB",
+          type: "GeneDescriptor",
+          label: "PDGFRB",
+          gene_id: "hgnc:8804",
+        },
+        element_genomic_start: {
+          id: "fusor.location_descriptor:NC_000005.10",
+          type: "LocationDescriptor",
+          label: "NC_000005.10",
+          location: {
+            type: "SequenceLocation",
+            sequence_id: "refseq:NC_000005.10",
+            interval: {
+              type: "SequenceInterval",
+              start: {
+                type: "Number",
+                value: 150125577,
+              },
+              end: {
+                type: "Number",
+                value: 150125578,
+              },
+            },
+          },
+        },
+      },
+    ],
+    regulatory_elements: [],
+  };
+
+const useStyles = makeStyles(() => ({
+  menuLink: {
+    marginBottom: "15px",
+  },
+  menuHeader: {
+    marginTop: "30px"
+  },
+}));
+
+interface AppMenuProps {
+    open?: boolean,
+    handleDemo?: any,
+}
+
+export default function AppMenu(props: AppMenuProps): React.ReactElement {
+  const classes = useStyles()
+  return (
+    <Drawer variant="permanent" open={props.open} anchor="left" className="menu-drawer">
+        <Box ml="10px">
+          <Link href="/" color="inherit"><h3>VICC Fusion Curation</h3></Link>
+            <Box className={`${classes.menuHeader} ${classes.menuLink}`}><Typography variant="h6" color="inherit"><b>Tools</b></Typography></Box>
+            <Box ml="10px">
+              <Box className={classes.menuLink}><Link href="/assayed-fusion" color="inherit">Assayed Fusion Tool</Link></Box>
+              <Box className={classes.menuLink}><Link href="/categorical-fusion" color="inherit">Categorical Fusion Tool</Link></Box>
+              <Box className={classes.menuLink}><Link href="/utilities"  color="inherit">Utilities</Link></Box>
+            </Box>
+          
+            <Box className={`${classes.menuHeader} ${classes.menuLink}`}><Typography variant="h6" color="inherit"><b>Resources</b></Typography></Box>
+            <Box ml="10px">
+              <Box className={classes.menuLink}><Link href="https://fusions.cancervariants.org/en/latest/" target="_blank" color="inherit">Fusions Home Page</Link></Box>
+              <Box className={classes.menuLink}><Link href="https://github.com/cancervariants/fusion-curation" target="_blank" color="inherit">Code Repository</Link></Box>
+              <Box className={classes.menuLink}><Link href="https://cancervariants.org/" target="_blank" color="inherit">VICC</Link></Box>
+            </Box>
+            <Button color="inherit" onClick={() => props.handleDemo(demoAssayedFusion)}>use assay demo</Button>
+            <Button color="inherit" onClick={() => props.handleDemo(demoCategoricalFusion)}>use categorical demo</Button>
```

Comment:
I would conditionally hide these buttons on landing page since they do not do anything. It also might make sense to stick them underneath the corresponding tool. If you don't want to conditionally hide, I would consider routing to the tool page with the demo data.

Reply:
I would also remove the all caps on these

Reply:
It might also make sense to move these buttons inside the tool display itself rather than on the side bar

Reply:
These are just temporary! I left the buttons there for now for ease of testing but once James adds more demo data, I will be removing these buttons and instead populate dropdowns on the respective tools. The issue is #110

Reply:
Also want to note that the work moving these to dropdowns was too complex to make part of this ticket, which is why I would like for it to be a future iteration and merge this in now for the sake of consolidating updates